WcaOnRails/app/helpers/notifications_helper.rb

```diff
@@ -67,6 +67,15 @@ def notifications_for_user(user)
       }
     end
 
+    user.delegated_competitions.visible.over
```

Comment:
I agree, it makes sense to only do this for competitions that actually ought to have happened (confirmed and visible, but competitions only become visible after being comfirmed, so it's enough to just check visible).